### PR TITLE
Update latest release to 3.12.4

### PR DESCRIPTION
**Pull request by unknwon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ All notable changes to Sourcegraph are documented in this file.
 ### Fixed
 
 - The `/.auth/saml/metadata` endpoint has been fixed. Previously it panicked if no encryption key was set.
-- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.12.3` will need to manually modify their `docker run` command to use `sourcegraph/server:3.12.3` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
+- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.12.4` will need to manually modify their `docker run` command to use `sourcegraph/server:3.12.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```
Comment (ggilmore) [inline CHANGELOG.md]: This changelog entry shouldn't have been updated to 3.12.4.

Comment (unknwon) [reply CHANGELOG.md]: Nice, I was just wondering what's going on here, thanks!

File: doc/index.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ We actively maintain the two most recent monthly releases of Sourcegraph.
 Upgrades should happen across consecutive minor versions of Sourcegraph. For example, if you are
 running Sourcegraph 3.1 and want to upgrade to 3.3, you should upgrade to 3.2 and then 3.3.
 
-> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.12.3`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
+> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.12.4`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
```
Comment (unknwon) [inline doc/index.md]: @ggilmore Is this another instance that we shouldn't be modifying but somehow we did incremental change since 3.8?

Comment (ggilmore) [reply doc/index.md]: @unknwon, yes I believe that this should just be `sourcegraph/server:3.8.2`

Comment (unknwon) [reply doc/index.md]: Thanks, fixed!

**Pull request merged**